jupyter_server/extension/application.py

```diff
@@ -211,6 +215,7 @@ def _prepare_config(self):
         """
         traits = self.class_own_traits().keys()
         self.extension_config = Config({t: getattr(self, t) for t in traits})
+        self.serverapp.web_app.settings['{}_config'.format(self.extension_name)] = self.extension_config
```

Comment:
Is this line needed? We already update settings in `_prepare_settings()`: https://github.com/jupyter/jupyter_server/blob/e1b4ce49acb776bcfa55526a4dc1bf4d413995e7/jupyter_server/extension/application.py#L230
which gets called after this method.